### PR TITLE
feat: emit withdrawn rewards events for `LiquidUnfarm` and `LiquidUnfarmAndWithdraw`

### PR DESCRIPTION
**Pull request by queencre**
…AndWithdraw

## Description

closes: #116 

## Tasks

- [x] ~Add `withdrawn_rewards` events for `LiquidUnfarm` and `LiquidUnfarmAndWithdraw`~
- [x] Remove `withdrawn_rewards` event in `LiquidFarm`
- [x] Update `06_events.md` to include added events 

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [ ] Appropriate labels applied
- [ ] Targeted PR against correct branch
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/v0.45.9/docs/building-modules/README.md).
- [ ] Wrote unit and integration
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added relevant `godoc` [comments](https://go.dev/blog/godoc).
- [ ] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Review `Codecov Report` in the comment section below once CI passes

### CODE REVIEW

File: x/liquidfarming/keeper/liquidfarm.go [hunk 1]
```diff
@@ -161,15 +161,16 @@ func (k Keeper) LiquidUnfarm(ctx sdk.Context, poolId uint64, farmer sdk.AccAddre
 			sdk.NewAttribute(types.AttributeKeyFarmer, farmer.String()),
 			sdk.NewAttribute(types.AttributeKeyUnfarmingCoin, unfarmingCoin.String()),
 			sdk.NewAttribute(types.AttributeKeyUnfarmedCoin, unfarmedCoin.String()),
+			sdk.NewAttribute(types.AttributeKeyWithdrawnRewards, withdrawnRewards.String()),
 		),
 	})
 
-	return unfarmedCoin, nil
+	return unfarmedCoin, sdk.Coins{}, nil
```
Comment (hallazzang) [inline x/liquidfarming/keeper/liquidfarm.go:168]: Shouldn't we return `withdrawnRewards` here?

**Review by hallazzang — CHANGES_REQUESTED** [review]

File: x/liquidfarming/keeper/liquidfarm.go (hunk 1 above)
Comment (kingcre) [inline x/liquidfarming/keeper/liquidfarm.go:168]:
```suggestion
	return unfarmedCoin, withdrawnRewards, nil
```

**Review by kingcre — CHANGES_REQUESTED** [review]

**Comment by queencre** [comment]
After having an internal discussion, the event is not a necessary information for farmers as well as client developers. Instead, it is decided to remove the currently emitting event in `LiquidFarm` msg.

**Review by kingcre — APPROVED** [review]
utACK

**Pull request merged**